scripts/docker/buildpack-deps/Dockerfile.ubuntu2004

```diff
@@ -22,7 +22,7 @@
 # (c) 2016-2019 solidity contributors.
 #------------------------------------------------------------------------------
 FROM buildpack-deps:focal AS base
-LABEL version="15"
+LABEL version="16"
```

Comment:
Why `ubuntu2004` and `ubuntu2004.clang` also need a version bump?

Reply:
To force them to rebuild: the z3 change here is implicit from the `libz3-static-dev` and `z3-static` packages below on line 40. These packages were updated yesterday with z3 4.11.2 in https://launchpad.net/~ethereum/+archive/ubuntu/cpp-build-deps/+packages.